doc.go

```diff
@@ -13,4 +13,4 @@
 // gonum.org/v1/plot/plotter package
 // which is documented here:
 // https://godoc.org/gonum.org/v1/plot/plotter
-package plot // import "gonum.org/v1/plot"
+package plot
```

Comment:
please leave this import comment.
it makes sure people import "plot" via the vanity import path `"gonum.org/v1/plot"`.